.github/workflows/create-bcr-entry.mjs

```diff
@@ -0,0 +1,224 @@
+import crypto from "crypto";
+import {
+  readFileSync,
+  mkdirSync,
+  existsSync,
+  copyFileSync,
+  writeFileSync,
+  createWriteStream,
+  unlink,
+} from "fs";
+import https from "https";
+import { resolve } from "path";
+
+/**
+ * Create a bcr entry for a new version of this repository.
+ *
+ * Usage: create-bcr-entry [project_path] [bcr_path] [owner_slash_repo] [tag]
+ *
+ *   project_path: path to the project's repository; should contain a
+ *      root level MODULE.bazel file and a .github/workflows/bcr folder
+ *      with templated bcr entry files.
+ *   bcr_path: path to the bcr repository
+ *   owner_slash_repo: the github owner/repository name of the project
+ *   tag: the github tag for this version, e.g., "v1.0.0" or "1.0.0"
+ *
+ */
+async function main(argv) {
+  if (argv.length !== 4) {
+    console.error(
+      "usage: create-bcr-entry [project_path] [bcr_path] [owner_slash_repo] [tag]"
+    );
+    process.exit(1);
+  }
+
+  const [projectPath, bcrPath, ownerSlashRepo, tag] = argv;
+  const version = getVersionFromTag(tag);
+
+  const moduleName = getModuleName(resolve(projectPath, "MODULE.bazel"));
+  const bcrTemplatesPath = resolve(projectPath, ".github/workflows/bcr");
+  const bcrEntryPath = resolve(bcrPath, "modules", moduleName);
+  const bcrVersionEntryPath = resolve(bcrEntryPath, version);
+
+  if (!existsSync(bcrEntryPath)) {
```

Comment:
If the method is async any reason not to use the `fs/promises` API?

Reply:
I'd rather not have copies of this file diverge between repos, and it's temporary, so I'm not too concerned about minor improvements like this.

Reply:
Yeah, if this is just copied then don't change anything. I didn't realize it was copied until looking through a bit more.